thirdparty/portmidi/pm_win/pmwinmm.c

```diff
@@ -1,12 +1,13 @@
 /* pmwinmm.c -- system specific definitions */
-

```

Comment:
Please avoid all pure whitespace changes, in order to keep the patch as small as possible